testutil/accumulator.go

```diff
@@ -388,6 +388,65 @@ func (a *Accumulator) AssertDoesNotContainsTaggedFields(
 	return
 }
 
+// Asserts that at least one of the metrics contains all the required tags
+func (a *Accumulator) AssertTagValuesInMeasurement(
```

Comment:
Instead of adding these new functions, use `acc.GetTelegrafMetrics()` and check for the expected metrics:
```
actual := acc.GetTelegrafMetrics()
// filter out unrelated metrics
testutil.RequireMetricsEqual(t, expected, actual, testutil.IgnoreTime())
```

Reply:
The reason why I was choosing this approach is being more specific in what I'm testing (in this case whether those tags are present of not), rather than comparing the whole metric and having to define the expected metric result in my test (filtering out unrelated metrics as you suggested will help for sure reducing the noise).

Another reason is that in general, I prefer having specific test messages (in this case we would get a failure stating if it was related to `docker_container_cpu`, `docker_container_net` or `docker_container_blkio`; if asserting the whole metric we would get to compare the whole metric to see what is wrong).

Anyway it's your call, if you think it's better to assert the whole metric (or your guidelines follow this approach and you want an homogeneous approach) I can remove this new function and change the test to assert the whole metric.

Reply:
Thanks for the feedback, I will refactor to follow the guidelines you provided :)
> I'm trying to phase out all of the other acc.Assert* functions as well
....
Sorry about all the inconsistent styling in the tests, I realize this makes it impossible to follow the style we prefer.

I was thinking that maybe we should include some comments regarding this on the contributing guidelines (state `acc.Assert*` functions will be deprecated and encourage the use of the whitelisted `testutil` functions that should be used )